clippy_lints/src/manual_div_ceil.rs

```diff
@@ -130,6 +130,22 @@ fn check_eq_expr(cx: &LateContext<'_>, lhs: &Expr<'_>, rhs: &Expr<'_>) -> bool {
     SpanlessEq::new(cx).eq_expr(lhs, rhs)
 }
 
+// Check if the literal contains type suffix like `7_i32` or `7u32`
+fn is_include_type_suffix(cx: &LateContext<'_>, expr: &Expr<'_>) -> bool {
+    let snip = snippet(cx, expr.span, "..");
+    let numeric_types = [
+        "i8", "i16", "i32", "i64", "i128", "isize", "u8", "u16", "u32", "u64", "u128", "usize",
+    ];
+
+    for ty in numeric_types {
+        if snip.ends_with(ty) {
+            return true;
+        }
+    }
```

Comment:
I'm not sure if there is a better way to determine if `Expr` is a literal with a type suffix...

Reply:
If you match on the expr and it's `ExprKind::Lit(LitKind::Int)`, you can check if the int literal type is (not) `LitIntType::Unsuffixed`

Reply:
Thank you! I changed to use `LitIntType::Unsuffixed` in https://github.com/rust-lang/rust-clippy/pull/13864/commits/de863026912d9fa86db10003825273697218a6e4.